src/main/java/de/pandemieduell/api/GamesController.java

```diff
@@ -0,0 +1,47 @@
+package de.pandemieduell.api;
+
+import de.pandemieduell.model.GameState;
+import de.pandemieduell.model.Round;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class GamesController {
+
+  @RequestMapping(value = "/registerUser", method = RequestMethod.POST, consumes = "text/plain")
+  public long registerUser(@RequestBody String username) {
+    return 0; // TODO implement
+  }
+
+  @RequestMapping(value = "/joinGame", method = RequestMethod.GET)
+  public long joinGame(@RequestParam("playerId") long playerId) {
+    return 0; // TODO implement
+  }
+
+  @RequestMapping(value = "games/{gameId}/currentRound", method = RequestMethod.GET)
+  public Round getCurrentRound(@PathVariable long gameId) {
```

Comment:
This could be removed if we go with my suggestion below to have one endpoint that gets all game state